Add Audio icon to Layout Editor

danielb987 opened the pull request:
This also includes the fix for #12569. The cause for issue #12569 was that the property `BeanNameModule` was missing.

dsand47 commented:
The Web Panels context menu items are only available when the Audio icon is in icon mode.  It is not available when the icon is in text mode, but the mouse clicks are still active.

danielb987 commented:
> For the vertical tool bar and floating tool box, the "Shape" item should be moved to its own row. This will reduce the width of each.

Fixed.

> The Web Panels context menu items are only available when the Audio icon is in icon mode. It is not available when the icon is in text mode, but the mouse clicks are still active.

Fixed.

danielb987 commented:
This PR is ready for review.
